dashboard/src/components/AppView/AppView.v2.tsx

```diff
@@ -207,9 +210,7 @@ export default function AppView({
         </div>
       </PageHeader>
 
-      {error && (
-        <Alert theme="danger">Unable to load the application. Received: {error.message}</Alert>
-      )}
+      {error && <Alert theme="danger">Found an error: {error.message}</Alert>}
```

Comment:
"Found an error" is a bit vague perhaps? I assume you changed this because "Unable to load the application" is not generic enough (or just incorrect)? Perhaps `The previous operation falied with: {error.message}`

Reply:
yes, the problem is that the error can appear either when loading the app or when upgrading/deleting/rollingback so it's not always after an "operation"